domains/obsnews.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "vivianmediagroup",
+        "email": "viv@joindefy.com"
+    },
+    "record": {
+        "CNAME": "vivianmediagroup.github.io"
```

Comment:
404 CNAME record.

Reply:
That's odd, my apologies for the inconvenience. I'm having trouble debugging the problem. When I entered `obsnews.is-a.dev` in Github Pages, it said it's "improperly configured". 
```
Your site's DNS settings are using a custom subdomain, obsnews.is-a.dev, that is set up as an A record. We recommend you change this to a CNAME record pointing to vivianmediagroup.github.io. For more information, see [documentation](https://docs.github.com/articles/setting-up-a-custom-domain-with-github-pages/) (InvalidARecordError).
```
However, for some reason, the CNAME record is 404 (not found). My apologies, I don't know how to debug this on my end. Is the problem on my end, and if so what do I need to do to resolve it?

Thank you, and my most sincere apologies for the inconvenience.